Code/monet_cyclegan/data_acquisition/augment.py

```diff
@@ -121,6 +135,30 @@ def save_augmented_image(input_path: str, output_dir: str) -> None:
     save_image(image=processed_image,
                output_path=f'{output_dir}/augmented-{filename}')
 
+def save_augmented_image_without_crop(input_path: str, output_dir: str) -> None:
+    """Save an augmented image to a file.
+
+    Args:
+        input_path: The path of the original image.
+        output_dir: The directory of the augmented image.
+    """
+
+    if not os.path.isfile(input_path):
+        raise FileNotFoundError(f'Could not find file "{input_path}".')
+
+    image = read_image(path=input_path,
+                       width=IMAGE_SIZE[0],
+                       height=IMAGE_SIZE[1],
+                       channels=CHANNELS)[0]
+
+    processed_image = augment_image_without_crop(image=image)
+    processed_image = tensor_to_image(image=processed_image)
+
+    filename = os.path.basename(input_path)
+
+    save_image(image=processed_image,
+               output_path=f'{output_dir}/augmented-{filename}')
+
```

Comment:
Same thing for augment_image_without_crop--we can just add parameters for the existing functions to include/exclude different augmentations.

Reply:
I added a crop function for this parameter and latest commit.